src/commands/createHttpFunction.ts

```diff
@@ -29,6 +29,7 @@ export async function createHttpFunction(context: IActionContext): Promise<void>
         suppressCreateProjectPrompt: true,
         templateId: 'HttpTrigger',
         languageFilter: /Python|C\#|^(Java|Type)Script$/i,
-        functionSettings: { authLevel: 'anonymous' }
+        functionSettings: { authLevel: 'anonymous' },
+        targetFramework: 'netcoreapp3.1'
```

Comment:
Can you add a comment in the code related to what you said on GitHub. Something like:

```suggestion
        targetFramework: 'netcoreapp3.1' // Will only work on functions api v1.4.0, but won't hurt on v1.3.0
```